client/components/ProductSlider.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+const faker = require("faker");
+import "../stylesheets/productSlider.scss";
+
+const ProductSlider = () => {
+	return (
+		<div className="slider-container">
```

Comment:
set up a linter for the group! Avoid these tabs/spaces conflicts so the commits are clearer